variables.tf

```diff
@@ -125,24 +125,36 @@ variable "initial_position" {
   type        = string
 }
 
-variable "byte_limit" {
+variable "buffer_byte_limit" {
   description = "The amount of bytes to buffer events before pushing them to Elasticsearch"
   default     = 1000000
   type        = number
 }
 
-variable "record_limit" {
+variable "buffer_record_limit" {
   description = "The number of events to buffer before pushing them to Elasticsearch"
   default     = 500
   type        = number
 }
 
-variable "time_limit_ms" {
+variable "buffer_time_limit_ms" {
```

Comment:
Do we need to rename these variables?  Worry that they are no longer consistent across the other KCL apps which all use the variable without `buffer` as a prefix.